integration-tests/pino/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const options = {}
+
+if (process.env.AGENT_PORT) {
+  options.port = process.env.AGENT_PORT
+}
+
+if (process.env.AGENT_URL) {
```

Comment:
You don't need this if-block at all, since you're not using that option in the tests.